### ISSUE TITLE
Migrate module compartmentalization changes from Hawking

### ISSUE THREAD

**Issue by naschorr** [naschorr/clipster#59]
The recent work on https://github.com/naschorr/hawking/tree/78-module-compartmentalization for Hawking affects some of the common functionality shared between the bots, so those changes should be implemented and accounted for here as well.

**Comment by naschorr** [comment]
Closing, no longer relevant after #64
